troubleshoot/monitoring/node-moves-outages.md

```diff
@@ -1,5 +1,5 @@
 ---
-navigation_title: Node moves and outages
+navigation_title: Node moves and HW failures
```

Comment:
I'd like a suggestion for the nav title, I'm thinking of `Node moves and system maintenance` better than HW failures.
WDYT?

cc: @kunisen , @shainaraskas

Reply:
```suggestion
navigation_title: Node moves and Hardware failures
```